### PR TITLE
Added license headers to scripts

### PR DESCRIPTION
**Pull request by IzabellaRaulin**
### Summary of changes:
 - added license headers to scripts

### Tests:
none needed

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -1,5 +1,22 @@
 #!/bin/bash -e
 
+#http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#
+#Copyright 2015 Intel Corporation
```
Comment (thomastaylor312) [inline scripts/build.sh]: Should this be `Copyright 2015-2016`?

Comment (IzabellaRaulin) [inline scripts/build.sh]: @andrzej-k, could you verify that? Thank You.

Comment (andrzej-k) [inline scripts/build.sh]: Just having 2015 in the license header is fine.

**Comment by katarzyna-z** [comment]
LGTM

**Pull request merged**